md/bonita-as-windows-service.md

```diff
@@ -20,7 +20,7 @@ Please verify the supported version from the Support page
 
 ![Tomcat home page](images/bonita-as-windows-service/tomcatHome.png)
 
-* **_Download_** the bundle BonitaSubscription-x.y.z-tomcat.zip (or BonitaCommunity-x.y.z-tomcat.zip for the community edition)
+* **_Download_** the bundle BonitaSubscription-x.y.z.zip (or BonitaCommunity-x.y.z-tomcat.zip for the community edition)
```

Comment:
BonitaCommunity-x.y.z-tomcat.zip  -> it stay tomcat in BonitaCommunity zip name

Reply:
@danila-m ?